rpiplay.cpp

```diff
@@ -36,13 +36,13 @@
 #define VERSION "1.2"
 
 #define DEFAULT_NAME "RPiPlay"
-#define DEFAULT_SHOW_BACKGROUND true
+#define DEFAULT_BACKGROUND 1
```

Comment:
Please make it default to always/on, to keep backwards-compatibility.

Reply:
It has been done.